src/routes/Header.svelte

```diff
@@ -17,7 +17,7 @@
 		</svg>
 		<ul>
 			<li aria-current={$page.url.pathname === '/' ? 'page' : undefined}>
-				<a href="/">My Home</a>
+				<a href="/">My Home Sweet Home</a>
```

Comment:
```suggestion
				<a href="/">My Home (not yours)</a>
```

this is not intended to be a welcoming website